src/fiber/channel.rs

```diff
@@ -1105,7 +1105,7 @@ where
                 tlc_id: tlc.id.into(),
                 amount: tlc.amount,
                 payment_hash: tlc.payment_hash,
-                expiry: tlc.lock_time,
```

Comment:
We are sending an unverified timestamp to the peer. This may be a time in the past.

Reply:
all expiry time related verification should be done in a new PR